### DISCUSSION TITLE
Для разных польхователей компоненты в мессенджере разные

### DISCUSSION THREAD

**Discussion by yanusa3** [VKCOM/VKUI#7339] [Q&A] [answered]
Здравствуйте, при работе с приложением ВК и отправкой сообщений наткнулся на такой момент, что для разных аккаунтов используются разные компоненты при отправке сообщений, для одного из пользователей 
ConvoMessageInfoWithoutBubbles
а для другого
im-mess
Получается, что у них разный интерфейс. Поделитесь, пожалуйста, зачем так сделано и от чего зависит?

**Comment by SevereCloud** [comment, marked as answer]
Привет, эти дискуссии предназначены для обсуждения библиотеки [VKUI](https://github.com/VKCOM/VKUI).

У пользователей может быть разный интерфейс для сравнения разных показателей. А называется такой процесс [A/B-тестированием](https://ru.wikipedia.org/wiki/A/B-%D1%82%D0%B5%D1%81%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)
